source/pages/Tokens/SyscoinImport.tsx

```diff
@@ -25,26 +25,27 @@ export const SyscoinImportToken = () => {
   );
 
   const nextStep = async ({ assetGuid }: { assetGuid: string }) => {
-    try {
-      setIsLoading(true);
-
-      const metadata = await getAsset(activeNetwork.url, assetGuid);
-
-      if (metadata && metadata.symbol) {
-        await controller.wallet.account.sys.saveTokenInfo({
-          ...metadata,
-          symbol: metadata.symbol ? atob(String(metadata.symbol)) : '',
-        });
-
-        setAdded(true);
-        setIsLoading(false);
-
-        return;
-      }
-    } catch (_error) {
-      setError(Boolean(_error));
+    setIsLoading(true);
+
+    const addTokenMethodResponse =
+      await controller.wallet.assets.sys.addSysDefaultToken(
+        assetGuid,
+        activeNetwork.url
+      );
+
+    if (
+      typeof addTokenMethodResponse === 'boolean' ||
```

Comment:
same comment as above. Also in those cases why did you remove the try-catch? Maybe it is cleaner to handle the error.